Lombiq.NodeJs.Extensions/scripts/handle-error.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @summary Helper functions to display errors in the format MSBuild can understand.
+ */
+
+const os = require('os');
+
+function handleErrorObjectInner(error, type, defaultCode) {
+    if (error === null || error === undefined) {
+        return handleErrorObjectInner(Error("Missing error argument"), 'error', 'META-ERROR');
```

Comment:
It's called meta, because it's an error that happens during the error. That's conceptually very different from an internal error. Also typically MSBuild error codes are all caps symbols without spaces.

Reply:
Well, it's also an internal error, because the error reporting tool threw an error. But let's keep META-ERROR. At least it's fun!